### DISCUSSION TITLE
add font to the software

### DISCUSSION THREAD

**Discussion by DalilaBA** [flxzt/rnote#726] [Q&A] [answered]
is it possible to add some fonts to the software ?

**Comment by maymage** [comment, marked as answer]
Rnote should detect the fonts installed on your system, from which you can select then as here 

![grafik](https://github.com/flxzt/rnote/assets/36572014/58d98944-22e2-45ac-b238-ee77fa991460)

**Reply by DalilaBA** [reply]
ahh yes it needed a reboot , thank you very very much :D
